# glutamate-transporters-isoforms

### Abstract
Glutamate transporters play a key role in central nervous system physiology in their role in excitatory neurotransmitter homeostasis. Accordingly, their malfunction results in various neurological conditions. Biological assemblies of the transporters, consisting of cyclic homotrimers, emerge as a crucial aspect of glutamate transporter modulation. Hence targeting heteromerization promises an effective approach for modulator design. On the other hand, the dynamic nature of transcription allows for the generation of transporter isoforms in a structurally distinct manner. In this study, the potential isoforms were identified through the analysis of computationally generated gene-centric isoform maps. The conserved features of isoform sequences were revealed by computational chemistry methods and following structural analysis of AlphaFold2 predictions. The truncated isoforms were further subjected to a wide range of docking analyses, molecular dynamics simulations, and comparative genomics studies. Energetic landscape analyses indicated an inhibitory feature of isoforms maintaining analogous interaction patterns, emphasizing the significance of structural shape in isoform interactions. The isoforms also exhibited a strong tendency for cross-reaction. Furthermore, the trimerization and self-assembly dynamics that are seen in glutamate transporters, explored through structural analysis and molecular docking, underscored its inherent protective roles against inhibitory effects of the truncated isoforms. Comparative genomics studies further elucidated the genetic plasticity and evolutionary conservation represented by truncated isoforms. Our computational studies on glutamate transporters offer insights into cellular signaling, neurotransmission, and potential drug targets for physiological or pathological processes.

### Key Words: 
Truncated isoforms, Membrane protein design, Protein 3D structural predictions, Water-soluble integral membrane proteins. 
â€ƒ
### Data availability 
The AlphaFold DB (https://alphafold.ebi.ac.uk), a database developed by DeepMind and the European Bioinformatics Institute (EMBL-EBI) at EMBL, is a repository for AlphaFold2 predictions, with over 200 million protein structures. Each statistical and computational analysis of this study, included with step-by-step instructions where possible, are publicly available to ensure repeatability.  Further information and requests for data should be directed to and will be fulfilled by S.Z.
